docs/get-started/huggingface.md

```diff
@@ -0,0 +1,28 @@
+# HuggingFace Hub Integration
+
+Sample Factory has integrations with HuggingFace Hub to push models with evaluation results and training metrics to the hub. 
+
+### Downloading Models
+
+To download models from the HuggingFace Hub, use `git lfs` to clone the repo into your `train_dir`
+
+1. Download `git lfs` with `sudo apt install git-lfs`
```

Comment:
@andrewzhang505 can you explain this part? What does git lfs have to do with anything? I'm curious

Also instruction step 3. says just `git clone ` without an actual path. Is this intended?

Reply:
Maybe where you first mention a repo, say "HuggingFace Hub repo".
I thought for a second that it has something to do with our repo.
<link to your huggingface repo> should be good too